src/solve/moi.jl

```diff
@@ -120,9 +120,11 @@ function __solve(prob::OptimizationProblem, opt::Union{MOI.AbstractOptimizer, MO
     if MOI.get(optimizer, MOI.ResultCount()) >= 1
         minimizer = MOI.get(optimizer, MOI.VariablePrimal(), θ)
         minimum = MOI.get(optimizer, MOI.ObjectiveValue())
+        ret = Symbol(string(MOI.get(optimizer, MOI.TerminationStatus())))
```

Comment:
We should make this match the "standard retcodes", though those need to be created first 😅

Reply:
Should we leave them like this for now? Especially for the `NLopt` interface there is no way for the user to retrieve the `ret` from the optimisation at the moment.

Reply:
I think it makes sense to keep the optimiser's `retcode` until we can think of a better solution

Reply:
Yes indeed, just worth mentioning that these retcodes will indeed change.

Reply:
I agree. Maybe it would make sense to take the most comprehensive return code list which from what I can see might be the from `MOI` and map return code of each optimiser to this. That means the user does not lose any information and if any of the other libraries extends their return codes it would be likely be possible to map it. Also `MOI` if I remember distinguishes between successful global and local convergence which would make sense for global interface like `GalacticOptim`. Should that interface discussion be moved somewhere else btw?

Reply:
It should be moved to an issue.